docusaurus.config.js

```diff
@@ -235,6 +235,12 @@ module.exports = {
       },
     // SEO Global Metadata
     metadata: [{name: 'keywords', content: 'sumo logic, documentation, tutorials, quickstarts'}],
+    announcementBar: {
+      id: 'announcementBar',
+      content: `RSAC 2024 - it’s game on for AI-driven DevSecOps - meet us at booth #6271 on May 6–9. 👉<a target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSeCdX9T4-ksh_LhCAu7_C1SG3WDnjTgoV1Tgy8VoxwAankcQw/viewform">Book a meeting</a>.`,
```

Comment:
```suggestion
      content: `RSAC 2024 is almost here. It’s game on for AI-driven DevSecOps! Meet us at booth #6271 on May 6–9.  👉 <a target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSeCdX9T4-ksh_LhCAu7_C1SG3WDnjTgoV1Tgy8VoxwAankcQw/viewform">Book a meeting</a>.`,
```

Reply:
@kimsauce, this is just a suggestion for an alternative presentation. Throw it away if you don't like it.

Reply:
I love it! As long as it fits, looks great to me. Thank you.